core/static/css/wf_website.css

```diff
@@ -103,8 +103,13 @@ p {
 }
 
 .btn-outline-primary {
-  border-color: #06669a;
-  color: #06669a;
+  border-color: #06669a !important;
```

Comment:
These CSS changes shouldn’t be in this PR. Make sure to start your branches from The ’main’ branch to ensure the  working branch is clean

Reply:
Yeah, I will correct that